src/AzureDevOpsPolicyConfigurator/Helpers/ConnectionProvider.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
```

Comment:
I would suggest to move helper classes to the root namespace to avoid additional usings